client/script.js

```diff
@@ -0,0 +1,65 @@
+
+const searchButton = document.querySelector('.submit')
+const form = document.querySelector('#search-form')
+
+  const  init= async (e) => {
+   
+        e.preventDefault();
+        
+        clearArtistList();
+        
+        const randomInput = e.target.text.value;
+
+        if(randomInput.toLowerCase() === "random")
+        {
+            getRandom();
+        }
+        else
+        {
+            try
+            {
+
+                const response = await fetch('http://localhost:3000/artists')
+    
+                const data = await response.json()
+    
+                setData(data)
+     
+            } catch (err) {
+                console.log(err)
+            }
+        }
+    }
+  
+function getRandom(){
```

Comment:
-Have the fetch results load onto a separate page or perhaps a seprate html page and link into the button so when someone clicks search it comes up with results related to the text search
-have a look at the onclick function(helped with our buttons)